server/controllers/userControllers.js

```diff
@@ -14,9 +14,7 @@ export const getUsers = async (req, res) => {
 
     res.status(404).json({ message: 'User not found' });
   } catch (error) {
-    res
```

Comment:
sending an error response is not needed when using error handling middlerware. please remove them.